contributions/2018/Philadelphia-PAWS.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: "R-Ladies Philadelphia for Philadelphia Animal Welfare Society (PAWS) | [March-September 2018]"
+author: "R-Ladies Philadelphia"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, fig.align="center")
+```
+
+## Groups Involved
+
+### Chapter
+<img src="http://rladiesphilly.org/img/phillylogo.png" width="80" style="float: right;"/> R-Ladies Philadelphia
+
+ - [http://rladiesphilly.org](http://rladiesphilly.org)
+ - [https://www.meetup.com/rladies-philly/](https://www.meetup.com/rladies-philly)
+ - [https://twitter.com/rladiesphilly](https://twitter.com/rladiesphilly)
+<p></p>
+
+
+### Organization
+<img src="https://www.facebook.com/173336599365294/photos/746845415347740/" width="80" style="float: right;"/> PAWS is a 501(c)3 non-profit organization dedicated to saving Philadelphia’s homeless and at-risk animals. PAWS is the city’s largest rescue partner and provider of low-cost, basic veterinary care for pet owners and rescue organizations that cannot otherwise access or afford it. Through its three no-kill shelters, foster care network, and special events, PAWS finds loving homes for thousands of animals each year. PAWS is working to make Philadelphia a no-kill city where every healthy and treatable pet is guaranteed a home.
```

Comment:
Change to: `<img src="https://scontent-amt2-1.xx.fbcdn.net/v/t1.0-9/1977275_746845415347740_1290989253_n.jpg?_nc_cat=106&oh=9be01e4eeef35e12d30373a771b9ff57&oe=5C1852C2" width="80" style="float: right;"/>`